client-js/main/client/composite-client.js

```diff
@@ -34,61 +42,100 @@ import {Client} from "./client";
  */
 export class CompositeClient extends Client {
 
-    constructor(querying, subscribing, commanding) {
-        super();
-
-        /**
-         * @type QueryingClient
-         *
-         * @private
-         */
-        this._querying = querying;
-
-        /**
-         * @type SubscribingClient
-         *
-         * @private
-         */
-        this._subscribing = subscribing;
-
-        /**
-         * @type CommandingClient
-         *
-         * @private
-         */
-        this._commanding = commanding;
-    }
-
-    newQuery() {
-        return this._querying.newQuery();
-    }
-
-    execute(query) {
-        return this._querying.execute(query);
-    }
-
-    fetch({entity: cls, byIds: ids}) {
-        return this._querying.fetch(cls, ids);
-    }
-
-    newTopic() {
-        return this._subscribing.newTopic();
-    }
+  constructor(querying, subscribing, commanding) {
```

Comment:
Same here, please split up this file into 2 or 3. I'd say that one class per file would be an adequate strategy.